### PR TITLE
Fix build on Windows

### PR DESCRIPTION
**Pull request by ath0mas**
Review 'install' and 'test' to make them pass on Windows.
 ~~Update auto-generated readmes and tests outputs.~~

### CODE REVIEW

File: packages/turf-inside/README.md [hunk 1]
```diff
@@ -13,53 +13,18 @@ be convex or concave. The function accounts for holes.
...
-};
-
-//=features
-
-var isInside1 = turf.inside(pt1, poly);
-//=isInside1
-
-var isInside2 = turf.inside(pt2, poly);
-//=isInside2
+var pt = point([-77, 44]);
```
Comment (tmcw) [inline packages/turf-inside/README.md]: Could you separate these changes? The Windows support is positive, but the examples are intended _not_ to use `point` and `polygon` and other global turf methods.

Comment (DenisCarriere) [inline packages/turf-inside/README.md]: Could we not exclude the `@turf/helpers` global methods?

Most of the README.md updates in this PR is from all the JSDocs updates.

Comment (ath0mas) [inline packages/turf-inside/README.md]: Fine. I remove commit on "updating readmes" (put in in another PR to be discussed separately)? I keep the 2 other commits in this PR?

Comment (DenisCarriere) [inline packages/turf-inside/README.md]: I'd suggest opening a new PR with only the windows fixes. Try to keep your PR related to your title and without mixing no-related commits.

**Comment by ath0mas** [comment]
Kept only commit directly related to Windows build

**Comment by DenisCarriere** [comment]
@tmcw those review comments you mentioned have been resolved in this PR.
:+1:
